### DISCUSSION TITLE
cockroad test data

### DISCUSSION THREAD

**Discussion by safaci2000** [go-jet/jet#401] [General]
It seems cockroach DB is VERY slow when dealing with so many INSERT statements for data loading.  Their community recommends using other formats 

Particularly the https://www.cockroachlabs.com/docs/stable/import-into argument.

Example:

```sql
IMPORT INTO mytable (id, name, age)
CSV DATA ('/data/mydata.csv');
```

would you be open to up having the data represented as CSV files for faster data loading?  Right now cockroach DB takes about 2-3 minutes to spin up a DB with all the data loaded.  

https://www.cockroachlabs.com/docs/stable/migrate-from-postgres

**Comment by go-jet** [comment]
If it is faster, I'm fine with loading from CSV files.
